### ISSUE TITLE
error converting file on Windows and ubuntu

### ISSUE THREAD

**Issue by AlbionS** [potree/PotreeConverter#469]
I was trying to convert a file with PotreeConverter2.0.2 and my windows was totaly blocked. i have 8 CPU and 16GRAM. The file size was 20Go.

Than I tried to do the same file but on ubuntu, this time I used wine to launch the PotreeConverter.exe since I was not able to install it on ubuntu. 
And at 78% I got this error on ubuntu while on Windows it just stops my pc :
```
virtual memory(total): 30
virtual memory(used): 28.7
virtual memory(available): 1.7
virtual memory(used by process): 130'022.3
virtual memory(highest used by process): 130'022.3
physical memory(total): 30.4
physical memory(available): 1.7
physical memory(used): 28.7
physical memory(used by process): 27.9
physical memory(highest used by process): 28.3
```

I can provide a bug file if needed if this is just not a memory problem because PotreeConverter normally does not go over the limit RAM, i've already tried with bigger files and it worked

**Comment by midnight-dev** [comment]
I think this needs more information. There isn't an obvious reason as to what happened just based on utilization. How many points? What LAS format? How many attributes & were there special extra attributes that the converter may have held on to (in memory)? What was the text output of the converter? What were the options you passed into the CLI? Did it fail at the same point and exhibit the same behavior on other machines? Does the same thing happen if you split the point cloud up in something like CloudCompare so that only half of the data needs to be processed?

This needs a lot more information. If you're still able to provide a file that is known to have this issue, please do.

**Comment by AlbionS** [comment]
@midnight-dev thank you for your time.
The data I have is confidential, I can upload it via a zip file with password and send the link and password via email to you @midnight-dev  and @m-schuetz if you agree to not share the file or any photos of it.

This is the only file that I posses and has this error. I hope that you can reproduce it also. I am using the latest PotreeConverter (2.0.2) to convert it.

**Comment by potree** [comment]
Please send it to me, I'll have a look at it, and won't share it of course. Please send the smallest file that causes this issue and make sure it's compressed (e.g. laszip)

Thanks.

**Comment by midnight-dev** [comment]
Of course. But I'd caution you to first try Markus. As the author of this and previous versions, he's going to have the best intuition here. Plus, I may not be able to test the conversion until Friday anyway. In the meantime, could you give us more details along the lines of what I mentioned in my last comment?

By the way, for such a large las file, zip and tar gzip may not be optimal. I'd recommend 7zip or WinRAR, as 7z & rar files can be significantly smaller than traditional zip archives.
